src/main/java/Events.java

```diff
@@ -0,0 +1,13 @@
+public class Events extends Task {
```

Comment:
All of the other classes (Task, Events, Todos Deadlines) are all very readable and within coding standards. Great Job!
Perhaps considering renaming classes to either all plural (Tasks) or all singular (Todo, Event, Deadline) for more uniformity